channeld/channeld.c

```diff
@@ -153,6 +153,14 @@ struct peer {
 
 	/* Make sure peer is live. */
 	struct timeabs last_recv;
+
+	/* It means the number of confirmations needed before LOCKIN.
```

Comment:
How about: `Additional confirmations need for local lockin` ?

Reply:
good description!